src/index.html

```diff
@@ -1,14 +1,116 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+
+    <meta
+
+      charset="UTF-8"
+
+    >
+    <meta
+
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+
+    >
+    <meta
+
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+
+    >
+    <title>
+
+      Moyo header
+
+    </title>
+    <link
+
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+
+    >
+    <link
+
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+
+    >
+    <link
+
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+
+    >
+    <link
+
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700;500&display=swap"
+      rel="stylesheet"
+
+     >
+    <link
+
+      rel="stylesheet"
+      href="./style.css"
+
+    >
+
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header  class="header">
+
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo-log"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#Laptops & Computers" data-qa="hover"
```

Comment:
Follow the checklist 
![image](https://github.com/mate-academy/layout_moyo-header/assets/99403198/71ec8b5f-4fc1-440e-b2f0-b5ccd2afdd01)